app/next-client-app/app/(protected)/scanreports/[id]/layout.tsx

```diff
@@ -45,6 +45,15 @@ export default async function ScanReportLayout({
     { name: "Downloads", slug: "downloads", iconName: "Download" },
   ];
 
+   {/* Pushed the Edit button to the Items above so it appears on same level */}
+  if (canEdit) {
+    items.push({
+      name: "Edit Scan Report Details",
```

Comment:
```suggestion
      name: "Edit Scan Report",
```
The new button on the nav bar seems to be dominant. And we may not want that.

Reply:
Fixed